### PR TITLE
Migrating to new csproj format

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

**Comment by SimonCropp** [comment]
u cant update the transport and acceptance tests?

**Comment by SzymonPobiega** [comment]
Can I do it? I thought the source packages are not supported in the new format.

**Comment by SimonCropp** [comment]
the new format doesnt support changing the files. if u dont need to change them u can update

**Comment by SzymonPobiega** [comment]
@SimonCropp 👍 . I'll convert them.

File: GitVersion.yml [hunk 1]
```diff
@@ -1,7 +1,2 @@
 assembly-versioning-scheme: Major
-next-version: 3.1
-branches:
-  develop:
-    tag: alpha
-  releases?[/-]:
-    tag: rc
\ No newline at end of file
+next-version: 3.1
```
Comment (SimonCropp) [inline GitVersion.yml]: u can still do a release convention. just not with a wildcard. eg https://github.com/Particular/NServiceBus.CastleWindsor/blob/develop/GitVersion.yml

File: src/CompatibilityTests/Facade_3.1/Facade_3.1.csproj [hunk 2]
```diff
@@ -1,76 +1,17 @@
...
   <ItemGroup>
-    <ProjectReference Include="..\..\NServiceBus.SqlServer.CompatibilityTests.Common\NServiceBus.SqlServer.CompatibilityTests.Common.csproj">
-      <Project>{531ab217-effe-428c-8ca0-a8a70c907992}</Project>
-      <Name>NServiceBus.SqlServer.CompatibilityTests.Common</Name>
-    </ProjectReference>
-    <ProjectReference Include="..\..\NServiceBus.SqlServer\NServiceBus.SqlServer.csproj">
-      <Project>{fa1193bf-325c-4201-bb78-484032e09809}</Project>
-      <Name>NServiceBus.SqlServer</Name>
-    </ProjectReference>
+    <Folder Include="Properties\" />
```
Comment (SimonCropp) [inline src/CompatibilityTests/Facade_3.1/Facade_3.1.csproj]: delete this

**Comment by SimonCropp** [comment]
@SzymonPobiega i took the liberty of fixing some approval tests stuff

**Comment by SimonCropp** [comment]
@Particular/sqlserver-transport-maintainers green build :)

File: src/NServiceBus.SqlServer.UnitTests/APIApprovals.Approve.approved.txt [hunk 3]
```diff
@@ -17,23 +17,19 @@ namespace NServiceBus.Transport.SQLServer.ConnectionStrings
     public class ConnectionInfo
     {
         public ConnectionInfo() { }
-        [System.ObsoleteAttribute(@"Multi-instance setup is supported for backwards compatibility and won't be supported from Version 4.0. To specify schema use `UseSpecificSchema()`, to specify connection string use `EnableLegacyMultiInstanceMode()`. The member currently throws a NotImplementedException. Will be removed in version 4.0.0.", true)]
+        [System.ObsoleteAttribute(@"Multi-instance setup is supported for backwards compatibility and won't be supported from Version 4.0. To specify schema use `UseSpecificSchema()`, to specify connection string use `EnableLegacyMultiInstanceMode()`. Will be treated as an error from version 3.0.0. The member currently throws a NotImplementedException. Will be removed in version 4.0.0.", false)]
```
Comment (tmasternak) [inline src/NServiceBus.SqlServer.UnitTests/APIApprovals.Approve.approved.txt]: Are those changes caused by new version of `Fody.Obsolete`?

**Comment by tmasternak** [comment]
One comment. Otherwise LGTM.

File: src/NServiceBus.SqlServer.UnitTests/APIApprovals.Approve.approved.txt (hunk 3 above)
Comment (SimonCropp) [inline src/NServiceBus.SqlServer.UnitTests/APIApprovals.Approve.approved.txt]: Yeah cleaned up the default obsolete message

**Comment by SzymonPobiega** [comment]
@tmasternak @SimonCropp I've upgraded the acceptance and transport tests projects. Strange thing though. In the acceptance tests I can still see "NSB.AcceptanceTests" folder in VisualStudio (with all the test files) while in the transport tests I can't see the test files in VS. In this normal?

**Comment by SimonCropp** [comment]
@SzymonPobiega that feature of VS is a bit buggy. try deleting bin/obj/.vs and opening the sln again

**Comment by SzymonPobiega** [comment]
@SimonCropp it helped. Thanks ❤️

**Pull request merged**